src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -61,6 +61,8 @@ object SbtGit {
       action(state, command +: args)
     }
 
+    val QuotedString: Parser[String] = (DQuoteClass ~> any.+.string.filter(!_.contains(DQuoteClass), _ => "Invalid quoted string") <~ DQuoteClass)
```

Comment:
Is there an option that pulls this from `DefaultParsers` or am I wrong in assuming that exists there?